plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -559,8 +560,7 @@ private void doCreateView(ConnectorSession session, SchemaTableName schemaViewNa
         Optional<com.amazonaws.services.glue.model.Table> existing = getTable(session, schemaViewName);
         if (existing.isPresent()) {
             if (!replace || !isPrestoView(firstNonNull(existing.get().getParameters(), ImmutableMap.of()))) {
-                // TODO: ViewAlreadyExists is misleading if the name is used by a table https://github.com/trinodb/trino/issues/10037
```

Comment:
https://github.com/trinodb/trino/issues/10037

Reply:
is this PR supposed to close that issue?

Reply:
I think Issue https://github.com/trinodb/trino/issues/10037 should have been closed by https://github.com/trinodb/trino/pull/10186.

This PR https://github.com/trinodb/trino/pull/15580 is follow-up TODO task for https://github.com/trinodb/trino/pull/11499/files#diff-0c09267905a13855768761575aee308f9a1773e6f3d59c6347de78fa8dbc1c02R422

Reply:
> I think Issue #10037 should have been closed by #10186.

the issue is currently open